### PR TITLE
add tutorial for yurt-app-manager 

### PR DESCRIPTION
**Pull request by kadisi**
Signed-off-by: kadisi <iamkadisi@163.com>

### Ⅰ. Describe what this PR does

add tutorial for yurt-app-manager  

ref #171 

### Ⅴ. Special notes for reviews

@Fei-Guo @charleszheng44 @rambohe-ch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ The major OpenYurt components consist of:
   reverse proxy to establish a secure network access between the cloud site control plane and the edge nodes 
   that are connected to the intranet.
 
+## Addons
+   Openyurt plans to offer a variety of addons specifically for edge computing scenarios in the future. The major Openyurt addons consist of:
+- **[Yurt-App-Manager](docs/tutorial/yurt-app-manager.md)** : A kubernetes Operator. It includes two Kubernetes CRD resources: *NodePool* and *UnitedDeployment*
+    - **[NodePool](docs/enhancements/20201211-nodepool_uniteddeployment.md)**:  Users can quickly learn which node pools or regions are in their cluster. User can also quickly see which nodes are in the node pool. User can uniformly type label, annotation, and TAINts on nodes under the node pool. This simplifies the operation and maintenance management of nodes. The nodes in the NodePool are ideally accessible to each other through the Intranet.
+    - **[UnitedDeployment](docs/enhancements/20201211-nodepool_uniteddeployment.md)**: User can use a template to deploy the application in a different node pool. This template can support one of Kubernetes deployment, Statefulset.
```
Comment (Fei-Guo) [inline README.md:68]: deploy an application in different node pools.

File: README.md [hunk 2]
```diff
@@ -61,6 +61,12 @@ The major OpenYurt components consist of:
   reverse proxy to establish a secure network access between the cloud site control plane and the edge nodes 
   that are connected to the intranet.
 
+## Addons
+   Openyurt plans to offer a variety of addons specifically for edge computing scenarios in the future. The major Openyurt addons consist of:
+- **[Yurt-App-Manager](docs/tutorial/yurt-app-manager.md)** : A kubernetes Operator. It includes two Kubernetes CRD resources: *NodePool* and *UnitedDeployment*
+    - **[NodePool](docs/enhancements/20201211-nodepool_uniteddeployment.md)**:  Users can quickly learn which node pools or regions are in their cluster. User can also quickly see which nodes are in the node pool. User can uniformly type label, annotation, and TAINts on nodes under the node pool. This simplifies the operation and maintenance management of nodes. The nodes in the NodePool are ideally accessible to each other through the Intranet.
```
Comment (Fei-Guo) [inline README.md:67]: TAINTts->taints
Intranet -> intranet

File: docs/tutorial/yurt-app-manager.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
+
+# Use yurt-app-manager to manage your edge node and workload
+
+In this tutorial, we will show how the yurt-app-manager helps users manage 
+there edge nodes and workload.
+Suppose you have a Kubernetes cluster in an Openyurt environment, or a native Kubernetes cluster with at least two nodes.
+
+## Install yurt-app-manager
+
+### 1. create and push yurt-app-manager image
```
Comment (Fei-Guo) [inline docs/tutorial/yurt-app-manager.md:10]: I suggest to skip the details of step 1 & 2. We could prepare ready images in a public repo and checkin the test yamls in the tutorial directory.

File: docs/tutorial/yurt-app-manager.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+kubectl get pod -n kube-system |grep yurt-app-manager
+```
+
+## How to Use
+
+The Examples of NodePool and UnitedDeployment are in `config/yurt-app-manager/samples/` directory
+
+### NodePool 
+
+Users can quickly learn which node pools or regions are in their cluster. You can also quickly see which nodes are in the node pool. User can uniformly type label, annotation, and TAINts on nodes under the node pool. This simplifies the operation and maintenance management of nodes. The nodes in the NodePool are ideally accessible to each other through the Intranet.
```
Comment (Fei-Guo) [inline docs/tutorial/yurt-app-manager.md:75]: You don't need to explain the node pool here.

File: docs/tutorial/yurt-app-manager.md [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+
+- 1 create an nodepool 
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: apps.openyurt.io/v1alpha1
+kind: NodePool
+metadata:
+  name: hangzhou
```
Comment (Fei-Guo) [inline docs/tutorial/yurt-app-manager.md:83]: How about using a neutral name, e.g., Pool-A ?

File: docs/tutorial/yurt-app-manager.md [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+$ kubectl get pod -l app=ud-test
+NAME                                  READY   STATUS    RESTARTS   AGE
+ud-test-edge-ttthd-5bbb4b8664-2f5ss   1/1     Running   0          10m
+ud-test-edge-ttthd-5bbb4b8664-8sjs7   1/1     Running   0          3m20s
+ud-test-edge-ttthd-5bbb4b8664-99dml   1/1     Running   0          10m
+ud-test-edge-ttthd-5bbb4b8664-dvb8s   1/1     Running   0          10m
+ud-test-edge-ttthd-5bbb4b8664-lxhgr   1/1     Running   0          3m20s
+ud-test-edge-ttthd-5bbb4b8664-zj8ls   1/1     Running   0          3m20s
+```
+
```
Comment (Fei-Guo) [inline docs/tutorial/yurt-app-manager.md:239]: Remove the empty lines in the end.

File: README.md [hunk 7]
```diff
@@ -61,6 +61,12 @@ The major OpenYurt components consist of:
   reverse proxy to establish a secure network access between the cloud site control plane and the edge nodes 
   that are connected to the intranet.
 
+## Addons
```
Comment (rambohe-ch) [inline README.md:64]: how about separate all components into cloud components and edge components, not add a Addons chapter?

File: docs/tutorial/yurt-app-manager.md (hunk 3 above)
Comment (kadisi) [reply docs/tutorial/yurt-app-manager.md]: done, I added an new file `yurt-app-manager-dev.md` just for developers.

**Review by Fei-Guo — APPROVED** [review]
LGTM.

I will polish the readme later.

**Pull request merged**